### PR TITLE
Update parallel-processing-expansion.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with english version

### CODE REVIEW

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 1]
```diff
@@ -368,6 +368,23 @@ Kubernetes accepts and runs the Jobs you created.
...
 
+<!--
+### Clean up {#cleanup-2}
+```shell
+# Remove the Jobs you created
+# Your cluster automatically cleans up their Pods
+kubectl delete job -l jobgroup=jobexample
+```
+-->
+### 清理
```
Comment (tengqm) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:379]: do NOT throw away the explicit anchor ID.

Comment (guzj11) [reply content/zh/docs/tasks/job/parallel-processing-expansion.md]: suggestion applied

Comment (guzj11) [reply content/zh/docs/tasks/job/parallel-processing-expansion.md]: the commits have been squashed into one commit

Comment (guzj11) [reply content/zh/docs/tasks/job/parallel-processing-expansion.md]: can this commit merged ？

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**